### DISCUSSION TITLE
Prediction all frames in a video and processed to 99% and error

### DISCUSSION THREAD

**Discussion by Leorio183** [talmolab/sleap#742] [Help!] [answered]
![image](https://user-images.githubusercontent.com/105135795/167285894-3c97a7d3-48d3-4694-acc5-31766baa12d1.png)
![image](https://user-images.githubusercontent.com/105135795/167285897-0d08fe51-5869-42b7-ac26-c8b086193b3a.png)
I tried several videos using my trained model. I can predict random 20 frames in a video, but when I predict all frames, error occurs when when processed to 99%.

**Comment by talmo** [comment, marked as answer]
Hi @Leorio183,

This is most likely an encoding error related to not being able to reliably seek to the end frames. Try out [the steps here](https://sleap.ai/help.html#does-my-data-need-to-be-in-a-particular-format) and let us know if that works for you.

Cheers,

Talmo

**Comment by Leorio183** [comment]
@talmo,
Thank you very much! I tried and solved following your suggestion! Another question, how to concert the prediction result to a new video file like .mp4 or .avi in the GUI interface?

Have a nice day,
Leorio183

**Comment by talmo** [comment]
Is **View** → **Render Video Clip with Instances...** what you're looking for?

**Comment by talmo** [comment]
Moved to discussions and marked the above suggestion as an answer but feel free to [ask another question](https://github.com/talmolab/sleap/discussions/categories/help) if you're still having issues.
